tools/autograd/gen_python_functions.py

```diff
@@ -228,7 +228,8 @@ def group_declarations_by_name(declarations, should_bind_fn):
 
 
 def get_type_default(declaration):
-    if declaration['name'].startswith('randperm'):
+    if declaration['name'].startswith('randperm') or \
```

Comment:
can you change it so that it is more explicit, i.e. `declaration['name'] == triu_indices or declaration['name'] == tril_indices`?